ClemBot.Bot/bot/cogs/guild_info_cog.py

```diff
@@ -0,0 +1,53 @@
+import arrow
+import asyncio
+import discord
+from discord.ext import commands
+
+import bot.extensions as ext
+from bot.consts import Colors
+
+class GuildInfoCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @ext.command()
+    @ext.long_help('Shows information on the current guild/Discord server.')
```

Comment:
```suggestion
    @ext.long_help('Shows information on the current Guild/Discord server.')
```

Reply:
Guild isn't capitalized by itself in the Discord API docs unless it's a title or at the start of a sentence.